examples/Components/forcefield/QuadBendingFEMForceField.scn

```diff
@@ -1,33 +1,38 @@
 <!-- Mechanical QuadbendingFEMForceField Example -->
-<Node name="root" gravity="0 0 -9.81" dt="0.02">
-    <RequiredPlugin name="SofaSparseSolver"/>
-    <RequiredPlugin name="SofaOpenglVisual"/>
-    <RequiredPlugin name="SofaMiscCollision"/>
+<Node name="root" gravity="-1 0 0" dt="0.02">
+    <RequiredPlugin pluginName="SofaSparseSolver"/>
+    <RequiredPlugin pluginName="SofaOpenglVisual"/>
+    <RequiredPlugin pluginName="SofaMiscCollision"/>
+    <RequiredPlugin pluginName='SofaBoundaryCondition'/>
+    <RequiredPlugin pluginName='SofaGeneralDeformable'/>
+    <RequiredPlugin pluginName='SofaImplicitOdeSolver'/>
+    <RequiredPlugin pluginName='SofaLoader'/>
+    <RequiredPlugin pluginName='SofaMiscFem'/>
 
-    <VisualStyle displayFlags="showVisual showForceFields" />
+
+    <VisualStyle displayFlags="showVisual showForceFields showBehaviorModels" />
     <DefaultPipeline name="CollisionPipeline" verbose="0" />
     <BruteForceDetection name="N2" />
     <DefaultContactManager name="collision response" response="default" />
     <MinProximityIntersection name="proximity" alarmDistance="0.5" contactDistance="0.33"/>
-    <CollisionGroup name="group"/>
 
     <Node name="skin" gravity="0 0 -9.81">
-        <EulerImplicitSolver name="cg_odesolver"  rayleighStiffness="0.1" rayleighMass="0.1" />
+        <EulerImplicitSolver name="cg_odesolver" />
         <CGLinearSolver iterations="25" name="linear solver" tolerance="1.0e-9" threshold="1.0e-9" />
         <MeshVTKLoader filename="mesh/test_quad.vtk" name="loader" />
-        <MechanicalObject src="@loader" name="DOFs" />
+        <MechanicalObject template="Vec3d" src="@loader" name="DOFs" />
         <QuadSetTopologyContainer src="@loader" name="topology" />
         <QuadSetTopologyModifier name="Modifier" />
-        <QuadSetTopologyAlgorithms name="topoalgo" template="Vec3d"/>
```

Comment:
I realize that, in the next release of SOFA, this component will be removed. Is there any way for me to keep this component?

Reply:
Hello @nhnhan92

I think this change was introduced by @epernod because there was alternative maybe he can tell more about it.
Otherwise, most of the time the decision are taken collectively and there is always a way to make things happen, if you really needs this one could you elaborate a bit more what the component was useful for your case ?

DM